CK2Plus_expanded/history/provinces/1577 - Cinarca.txt

```diff
@@ -1,25 +1,25 @@
-# 1600 - Cinarca
-
-# County Title
-title = c_cinarca
-
-# Settlements
-max_settlements = 3
-b_cinarca = castle
-b_ajaccio = city
-b_bonifacio = temple
-#b_sartene = city
-#b_portevecchio = castle
-#b_filitosa = castle
-#b_bastelicaccia = temple
-#b_propriano = castle
-
-# Misc
-culture = corsican
-religion = catholic
-terrain = mountain
-
-# History
-1077.1.1 = { trade_post = b_della_gherardesca }
-1100.1.1 = { culture = italian }
-1195.1.1 = { trade_post = b_spinola }
+# 1600 - Cinarca
+
+# County Title
+title = c_cinarca
+
+# Settlements
+max_settlements = 3
+b_cinarca = castle
+b_ajaccio = city
+b_bonifacio = temple
+#b_sartene = city
+#b_portevecchio = castle
+#b_filitosa = castle
+#b_bastelicaccia = temple
+#b_propriano = castle
+
+# Misc
+culture = italian #sardinian
+religion = catholic
+terrain = mountain

```

Comment:
Also, as noted before: Please remove any occurrences of "terrain = XY" from province history pending a potential future revisit.

Reply:
I wanted to do that after we are done assigning the provinces.

Reply:
Okay update: Triumvirate tells me this is not important, as it can be done by "regular expressions".
So you don't need to do it manually, we can do it automated for all provinces at some later point.